### DISCUSSION TITLE
V2.9 ETA

### DISCUSSION THREAD

**Discussion by stuartmcgillivray** [elsa-workflows/elsa-core#3239] [Q&A] [answered]
Hi,

I was curious what's the eta for v2.9?

I am aware preview releases of 2.9 are available via myget but I would like to know when the next stable release would likely be available.

Thanks!

**Comment by sfmskywalker** [comment, marked as answer]
Hi @stuartmcgillivray, I am planning on releasing 2.9 this weekend or during the week thereafter (so October 9 at the latest).

**Reply by maltegoetz** [reply]
@sfmskywalker Is there any update on this? I noticed that the milestone has been adjusted so that everything is complete now, but there does not seem to be a release yet.
